leftwm/src/bin/leftwm.rs

```diff
@@ -133,28 +133,30 @@ fn start_leftwm() {
 
     set_env_vars();
 
-    // Boot everything WM agnostic or LeftWM related in ~/.config/autostart
-    let mut children = Nanny::autostart();
-
-    let flag = get_sigchld_flag();
-    let mut leftwm_session = start_leftwm_session(&current_exe);
-    while leftwm_is_still_running(&mut leftwm_session) {
-        // remove all child processes which finished
-        children.remove_finished_children();
-
-        while is_suspending(&flag) {
-            nix::unistd::pause();
+    loop {
+        // Boot everything WM agnostic or LeftWM related in ~/.config/autostart
+        let mut children = Nanny::autostart();
+
+        let flag = get_sigchld_flag();
```

Comment:
loop should start after here, as the children and flag dont need resetting

Reply:
Thanks.